t/tags_unit.t

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/perl -w
+
+use Modern::Perl '2017';
+use utf8;
+
+use Test::MockModule;
+use Test::More;
+
+use ProductOpener::Tags qw/get_lc_tagid/;
+
+=head1 Some unit tests for Tags.pm module
+
+=cut
+
+=head2 Unit testing get_lc_tagid
+=cut
+
+# a fake remove stopwords
+sub fake_remove_stopwords($$$) {
+
+	my $tagtype = shift;
+	my $lc = shift;
+	my $tagid = shift;
+
+    # naivly remove "the" at start
+    $tagid =~ s/the-*//i;
+
+    return $tagid;
+}
+
+{
+    my $tag_module = Test::MockModule->new('ProductOpener::Tags');
+    # mock download image to fetch image in inputs_dir
+    $tag_module->mock('remove_stopwords', \&fake_remove_stopwords);
+
+    my $synonyms_ref = {
+        "en" => {
+            "salted-snacks" => "en:salted-snacks",
+            "salty-snacks" => "en:salted-snacks",
+            "salty-snack" => "en:salted-snacks",
+        }
+    };
+
+    my $lc_tagid;
+
+    $lc_tagid = get_lc_tagid($synonyms_ref, "en", "category", "salted-snacks", "");
```

Comment:
Not very important, but "category" should be "categories".

Reply:
ok, better stay close to reality, it helps understand better.